zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command/UserTaskListenersTest.java

```diff
@@ -103,4 +81,83 @@ void shouldCompleteUserTaskWithCompleteTaskListener() {
           assertThat(userTask.getAction()).isEqualTo("complete");
         });
   }
+
+  @Test
+  void shouldRejectCompleteTaskListenerJobCompletionWhenVariablesAreSetAndCreateIncident() {
```

Comment:
💭 This test looks quite complex to me, it might be beneficial to have a comment with high-level description of what the test does to improve the readability (the comments for the last two blocks of code are very useful).

Reply:
Thanks, Ana! I've added a comment for this test to clarify its purpose and outline the expected behaviour.